OrbitGl/CaptureSerializer.cpp

```diff
@@ -72,15 +72,15 @@ void CaptureSerializer::SaveImpl(T& archive) {
 
   {
     ORBIT_SIZE_SCOPE("Functions");
-    std::vector<Function> functions;
+    std::vector<protos::FunctionInfo> functions;
     for (auto& pair : Capture::GSelectedFunctionsMap) {
-      Function* func = pair.second;
+      protos::FunctionInfo* func = pair.second;
       if (func != nullptr) {
         functions.push_back(*func);
       }
     }
 
-    archive(functions);
+    // archive(functions);
```

Comment:
Why is this commented?

Reply:
I removed all this stuff in c431424dc4d4d772a8585c33631ceb205c8eb9c7 when added serialization with protobuf, but it's still a part of the history. I'll update commits after the review.